content/post/career-ladders-part-1.md

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
@ilanman should you rebase master to not accidentally deploy these?

Reply:
Github is giving me some trouble. Will probably squash my commits and just commit the latest change.